src/main/java/com/auth0/json/mgmt/EmailTemplate.java

```diff
@@ -0,0 +1,190 @@
+package com.auth0.json.mgmt;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Class that represents an Auth0 Guardian Factor object. Related to the {@link com.auth0.client.mgmt.GuardianEntity} entity.
+ */
+@SuppressWarnings({"unused", "WeakerAccess"})
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class EmailTemplate {
+
+    @JsonProperty("template")
+    private String name;
+    @JsonProperty("body")
+    private String body;
+    @JsonProperty("from")
+    private String from;
+    @JsonProperty("resultUrl")
+    private String resultUrl;
+    @JsonProperty("subject")
+    private String subject;
+    @JsonProperty("syntax")
+    private String syntax;
```

Comment:
should this be an enum for additional validation? Although only supported value is `liquid`

Reply:
HMMM. I discussed this with the server guys. The only allowed value is `liquid` but they say the option is only there for backwards compatibility, so we shouldn't be hardcoding it to `liquid` and lowering the param count for example. I don't think offering an enum/constant helps but I don't mind adding one. Would you prefer me to add it?

Reply:
Can you set the param default to `liquid` ? Then they don't need to think about it but in the future if they need to, they can change.